workspace_tools/targets.py

```diff
@@ -651,6 +651,16 @@ def __init__(self):
         self.default_toolchain = "uARM"
         self.supported_form_factors = ["ARDUINO", "MORPHO"]
         self.detect_code = ["0740"]
+        
+class ELMO_F411RE(Target):
+    def __init__(self):
+        Target.__init__(self)
+        self.core = "Cortex-M4F"
+        self.extra_labels = ['STM', 'STM32F4', 'STM32F411RE']
+        self.supported_toolchains = [ "GCC_ARM" ]
+        self.default_toolchain = "GCC_ARM"
```

Comment:
default toolchain - ARM or uARM - it's related to the online compiler.

I see there are uARM/ARM target files in this PR. Why supported toolchain is only GCC_ARM?